services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerClusterConfig.java

```diff
@@ -1767,6 +1770,10 @@ public int getDanglingTopicOccurrenceThresholdForCleanup() {
     return danglingTopicOccurrenceThresholdForCleanup;
   }
 
+  public boolean isHybridStorePartitionCountUpdateEnabled() {
+    return updateRealTimeTopic;
```

Comment:
Same comment again: let's keep var names and method name consistent please. Thanks